src/NodaTime/TimeZones/ZoneRecurrence.cs

```diff
@@ -167,11 +157,9 @@ public bool Equals(ZoneRecurrence other)
             }
 
             // We've got a transition earlier than we were asked for. Try next year.
+            // Note that this will stil be within the FromYear/ToYear range, otherwise
```

Comment:
This seems correct to me, though I'll admit that I'm more convinced by our existing test coverage.

Reply:
Not sure what you mean by the latter part - do you mean you're convinced by me being able to cover the lines I've now removed?